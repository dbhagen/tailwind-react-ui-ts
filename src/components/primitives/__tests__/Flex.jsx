import React from 'react'
import { shallow } from 'enzyme'

import { Flex } from '..'

const setup = (testProps = {}) => {
  const props = Object.assign({}, testProps)

  const wrapper = shallow(<Flex {...props}>Hello World</Flex>)

  return {
    props,
    wrapper,
  }
}

describe('Flex', () => {
  it('renders matching snapshot', () => {
    const { wrapper } = setup()

    expect(wrapper).toMatchSnapshot()
  })
})
