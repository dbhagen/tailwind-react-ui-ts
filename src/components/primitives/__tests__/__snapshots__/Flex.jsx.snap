// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Flex renders matching snapshot 1`] = `
<Box
  flex={
    Array [
      true,
    ]
  }
  inline={false}
  inlineBlock={false}
  inlineFlex={false}
  is="div"
>
  Hello World
</Box>
`;
